nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -724,8 +724,16 @@ in
       # commented, because can cause extra delays during activate for this config:
       #      services.nginx.virtualHosts."_".locations."/".proxyPass = "http://blabla:3000";
       # stopIfChanged = false;
-      serviceConfig.Type = "oneshot";
-      serviceConfig.TimeoutSec = 60;
+      serviceConfig = {
+        Type = "oneshot";
+        TimeoutSec = 60;
+        # User and group
+        User = cfg.user;
```

Comment:
@flokli why/how does `cfg.user` have permission to run `${pkgs.systemd}/bin/systemctl reload nginx.service`? :thinking: